src/sass/components/_receiver_badge.scss

```diff
@@ -21,51 +21,45 @@
         }
     }
 }
-.receiver-badge {
-  display: flex;
-  flex-direction: row;
-  align-items: center !important;
-  width: 100%;
 
-  .avatar-box {
-    margin-right: $main-padding;
-  }
+.receiver-badge {
+    display: flex;
+    flex-direction: row;
+    align-items: center !important;
+    width: 100%;
 
+    .avatar-box {
+        margin-right: $main-padding;
+    }
 
-  .receiver-badge-name {
-    margin-right: $main-padding;
-  }
-  /**
-    Contact badge specific rules
-   */
-  &.contact-badge {
+    .receiver-badge-name {
+        margin-right: $main-padding;
+    }
 
-    .contact-badge-identity {
-      flex: 1;
-      text-align: right;
-      font-size: 10pt;
-      color: #757575;
+    // Contact badge specific rules
```

Comment:
This comment is a bit redundant, given the class name :)

Reply:
Not mine but I can remove it. :grin: